src/io/perun.clj

```diff
@@ -101,20 +122,21 @@
    t target-key TARGET-PROP kw "Property name to store gravatar url"]
   (let [pod (create-pod ttr-deps)]
     (boot/with-pre-wrap fileset
-      (let [files (perun/get-meta fileset)
+      (let [files         (perun/get-meta fileset)
             updated-files (pod/with-call-in @pod
                             (io.perun.gravatar/find-gravatar ~files ~source-key ~target-key))
             fs-with-meta  (perun/set-meta fileset updated-files)]
         (u/dbug "Find gravatars:\n%s\n"
-                (pr-str (map target-key (vals updated-files))))
+                (pr-str (map target-key updated-files)))
       fs-with-meta))))
 
+;; Should be handled by more generic filterer options to other tasks
 (deftask draft
   "Exclude draft files"
   []
   (boot/with-pre-wrap fileset
     (let [files         (perun/get-meta fileset)
-          updated-files (perun/filter-vals #(not (true? (:draft %))) files)
+          updated-files (filter #(not (true? (:draft %))) files)
           fs-with-meta  (perun/set-meta fileset updated-files)]
       (u/info "Remove draft files. Remaining %s files\n" (count updated-files))
       fs-with-meta)))
```

Comment:
`draft` task makes and inpact on all downstream tasks not only `render`. E.x. we don't want to include draft posts into the sitemaps etc. It's something that should be done on the top of the tasks chain and not in the bottom.

Maybe we can create more general `filter` task instead of `draft` but that is another question